### DISCUSSION TITLE
Spinners are cheeseable by binding don and kat to same key (osu!taiko)

### DISCUSSION THREAD

**Discussion by Non-Binary-Programmer** [ppy/osu#24763] [Q&A] [answered]
If one key is bound to both a rim hit and a center hit, then hitting that key will always advance a spinner by two keypresses. The most practical way to do this without external programs would be to bind the built-in second input for the left side center and rim inputs to the same key, and use the remaining 4 inputs regularly. This might be problematic, but doesn't give an advantage over someone who can properly alternate dkdk with both hands. However, by using external programs, or by sacrificing your regular inputs for the right side center and rim, you can hit all four with one key, allowing you to advance the spinner by 4 hits with one keypress, and generating a major advantage.

Is this intentional?

**Comment by peppy** [comment, marked as answer]
Tracking at https://github.com/ppy/osu/issues/24780.
